src/options/getPackageOptions.ts

```diff
@@ -12,7 +12,7 @@ import path from 'path';
 export function getCombinedPackageOptions(actualPackageOptions: Partial<PackageOptions>): PackageOptions {
   const defaultOptions = getDefaultOptions();
   const cliOptions = getCliOptions(process.argv);
-  const rootOptions = getRootOptions(cliOptions);
+  const rootOptions = getRepoOptions(cliOptions);
```

Comment:
This is entirely a stylistic nit, and as such, feel free to ignore. Here goes. To be extra consistent, do you think it would make sense to also rename this local binding to `repoOptions`? The two immediate benefits I can see are 1) consistency of the rename, and 2) less confusion for other potential contributors when browsing the sources.

```suggestion
  const repoOptions = getRepoOptions(cliOptions);
```

Reply:
Good point, I just missed that since I did the function name change from another file.